### DISCUSSION TITLE
Issues with opaque in Identify.

### DISCUSSION THREAD

**Discussion by kiloreux** [ImageMagick/ImageMagick#3433] [Help] [answered]
Previously when running `identify -format "%[opaque]" sample-file.jpg` I get something like `True` in output. Which makes perfect sense. 

However in the latest version 7.0.11-4, if I run the same command. I get something like `FalseFalse` or `TrueFalse` which is a bit confusing. Is this broken? Or is this intended? 

Original file here https://drive.google.com/file/d/1K_44EPTD1Fpk0ov1SlxfCU-AqB--NqoF/view?usp=sharing

Thank you for your help.

**Comment by dlemstra** [comment]
Can you please share the test image you used so we can provide you with an answer?

**Reply by kiloreux** [reply]
Apologies. 

Here is the file I am using https://drive.google.com/file/d/1K_44EPTD1Fpk0ov1SlxfCU-AqB--NqoF/view?usp=sharing

**Comment by snibgo** [comment]
I expect your input had two images, so you get one value for each image.

**Reply by kiloreux** [reply]
That seems to be a breaking change since it wasn't the case for previous versions.

**Comment by snibgo** [comment]
As far as I know, this has always been the behaviour. For example, with v6.9.9-50:
```
convert xc: xc: -format "%[opaque]" info:
truetrue
```
It is sensible to use "\n" to divide the output into lines:
```
convert xc: xc: -format "%[opaque]\n" info:
true
true
```

**Reply by kiloreux** [reply]
Yes, except in the file I provided. The previous behavior with `7.0.4-10`

Would give this on the file `identify -format %n 9999716699CY.EPS` >>> 1

And now same command gives `identify -format %n 9999716699CY.EPS` >>> 2 

So it has changed from detecting only one page on the file to two.

**Comment by snibgo** [comment]
That would seem to be a bug. What does "magick 9999716699CY.EPS info:" say?

**Reply by kiloreux** [reply]
In the latest version it says this:

```
9999716699CY.EPS[0] EPT 400x260 400x260+0+0 16-bit ColorSeparation CMYK 416101B 0.020u 0:00.013
9999716699CY.EPS[1] TIFF 401x261 401x261+0+0 8-bit Palette sRGB 256c 211279B 0.020u 0:00.009
```

In the older version, it says this:

`9999716699CY.EPS EPT 400x260 400x260+0+0 16-bit ColorSeparation CMYK 416KB 0.000u 0:00.000`

Not sure what this means @snibgo .

**Comment by snibgo** [comment, marked as answer]
It means the older version could not see the TIFF image embedded within the Postscript file. But the new version can see this.

So this isn't a bug, but a new feature.

**Reply by kiloreux** [reply]
Thank you :pray:
